app/Libraries/UKCP.php

```diff
@@ -140,4 +144,38 @@ public static function getPathForToken($tokenID, $account)
     {
         return self::TOKEN_PATH_ROOT.$account->id.'/'.$tokenID.'.json';
     }
+
+    public function getStandStatus(string $airfield): array
```

Comment:
Is it worth being explicit that we are after an ICAO here in the variable name?

Reply:
Yes! fb85e2ea